app/logic/solana_logic.rb

```diff
@@ -73,27 +73,10 @@ def validators_cli
       raise 'No results from `solana validators`' if validators.nil?
 
       raise 'No results from `solana validators`' if \
-        validators['currentValidators'].empty? &&
-        validators['delinquentValidators'].empty?
+        validators['validators'].empty?
```

Comment:
You can use `validators['validators'].blank?` for checking both `#nil?` and `#empty?`